### PR TITLE
All the TDS are rounded to 2 decimal poimts

### PR DESCRIPTION
**Pull request by gautam071**
### Description
The TDS values are rounded to 2 decimal points in the monthly tax report

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Invoice/Resources/views/tax-report.blade.php [hunk 1]
```diff
@@ -47,10 +47,14 @@
...
-                        <td>{{ $invoice->tds }}</td>
+                        <td>{!! number_format((float)($invoice->tds), 2) !!}</td>
                         <td>{{ $invoice->sent_on->format(config('invoice.default-date-format')) }}</td>
                         <td>{{ $invoice->payment_at ? $invoice->payment_at->format(config('invoice.default-date-format')) : '-'  }}</td>
-                        <td> {{ Str::studly($invoice->status) }}</td>
+                        @if($invoice->status == 'paid')
+                            <td><p class="text-success">{{ Str::studly($invoice->status) }}</p></td>
+                        @else
+                            <td> {{ Str::studly($invoice->status) }}</td>
+                        @endif
```
Comment (GauravGusain98) [inline Modules/Invoice/Resources/views/tax-report.blade.php:57]: same as above

File: Modules/Invoice/Resources/views/tax-report.blade.php [hunk 2]
```diff
@@ -83,7 +87,11 @@
                         <td>{{ $invoice->conversion_rate_diff }}</td>
                         <td>{{ $invoice->sent_on->format(config('invoice.default-date-format')) }}</td>
                         <td>{{ $invoice->payment_at ? $invoice->payment_at->format(config('invoice.default-date-format')) : '-'  }}</td>
-                        <td> {{ Str::studly($invoice->status) }}</td>
+                        @if($invoice->status == 'paid')
+                            <td><b><p class="text-success">{{ Str::studly($invoice->status) }}</p></b></td>
+                        @else
+                            <td> {{ Str::studly($invoice->status) }}</td>
+                        @endif
```
Comment (GauravGusain98) [inline Modules/Invoice/Resources/views/tax-report.blade.php:94]: same here

File: Modules/Invoice/Services/InvoiceService.php [hunk 3]
```diff
@@ -248,13 +248,13 @@ private function formatInvoicesForExportAll($invoices)
...
-                'Amount' => $invoice->display_amount,
+                'Amount' => (float) str_replace(['$', '₹'], '', $invoice->display_amount),
                 'GST' => $invoice->gst,
-                'Amount (+GST)' => $invoice->invoiceAmount(),
+                'Amount (+GST)' => (float) str_replace(['$', '₹'], '', $invoice->invoiceAmount()),
                 'Received amount' => $invoice->amount_paid,
                 'Bank Charges' => $invoice->bank_charges,
                 'Conversion Rate Diff' => $invoice->conversion_rate_diff,
-                'TDS' => $invoice->tds,
+                'TDS' => number_format((float) ($invoice->tds), 2),
```
Comment (GauravGusain98) [inline Modules/Invoice/Services/InvoiceService.php:257]: Why these changes are here?

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments and also please link the issue id in the description. I am not able to find the issue for which this PR is made.

**Comment by GauravGusain98** [comment]
@gautam071 Please check the comments and resolve the merge conflicts.

File: Modules/Invoice/Resources/views/index.blade.php [hunk 4]
```diff
@@ -51,7 +51,11 @@
                     <td class = '{{ $invoice->shouldHighlighted() ? 'font-weight-bold text-danger ' : ''}}'>
                         {{ $invoice->receivable_date->format(config('invoice.default-date-format'))  }}
                     </td>
-                    <td> {{ Str::studly($invoice->status)  }}</td>
+                    @if($invoice->status == 'paid')
+                        <td><b><p class="text-success">{{ Str::studly($invoice->status) }}</p></b></td>
+                    @else
+                        <td> {{ Str::studly($invoice->status) }}</td>
+                    @endif
```
[the remark this answers is not in the document]
Comment (gautam071) [reply Modules/Invoice/Resources/views/index.blade.php:58]: This is not in this file i will change in that file

**Comment by GauravGusain98** [comment]
The merge conflicts are not yet resolved.

**Comment by gautam071** [comment]
> The merge conflicts are not yet resolved.

the conflicts are resolved

**Review by GauravGusain98 — APPROVED** [review]
Looks good

**Pull request merged**